enterprise/backup/src/main/java/org/neo4j/backup/impl/OnlineBackupContextBuilder.java

```diff
@@ -191,12 +191,25 @@ public OnlineBackupContext createContext( String... args ) throws IncorrectUsage
         {
             throw new IncorrectUsage( e.getMessage() );
         }
-        catch ( UncheckedIOException | IOException e )
+        catch ( UncheckedIOException e )
         {
             throw new CommandFailed( e.getMessage(), e );
         }
     }
 
+    private Path getFolder( Arguments arguments ) throws CommandFailed
+    {
+        Path path = arguments.getMandatoryPath( ARG_NAME_BACKUP_DIRECTORY );
+        try
+        {
+            return path.toRealPath();
+        }
+        catch ( IOException e )
+        {
+            throw new CommandFailed( String.format( "Directory '%s' does not exist.", path.toFile().toString() ) );
```

Comment:
The `toString` from `Path` should do the same, and will be implicitly called on any object processed by `%s`.